external/fv3fit/fv3fit/_shared/models.py

```diff
@@ -281,7 +284,9 @@ def __init__(
 
     def predict(self, X: xr.Dataset) -> xr.Dataset:
         base_predict = self.base_model.predict(X)
-        novelties = self.novelty_detector.predict_novelties(X, self.cutoff)
+        novelties = self.novelty_detector.predict_novelties(
+            X, cutoff=self.cutoff, taper_function=self.taper_function
+        )
```

Comment:
(see comment in NoveltyDetector about moving the taper function call out to here)
I think you should keep the previous behavior of having `self.novelty_detector.predict_novelties(X, self.cutoff)` just return information about novelty score and label, and have a separate call after that to 
```
taper_values = self.taper_function(novelties[whatever_the_score_var_is])
```
You can merge that into the final dataset at the end.

The one part that looks like it is NoveltyDetector dependent is the default cutoff- I think you could leave the `get_default_cutoff` method on the NoveltyDetectors, the usage for setting up the default taper functions in their .load looks fine.